CoreVulkanDisplay/src/au/gov/asd/tac/constellation/visual/vulkan/renderables/CVKAxesRenderable.java

```diff
@@ -294,8 +332,9 @@ public static int StaticInitialise(CVKDevice cvkDevice) {
         int ret = VK_SUCCESS;
         if (!staticInitialised) {
             LoadShaders(cvkDevice);
-            if (VkFailed(ret)) { return ret; }
-            //ret = CreateDescriptorLayout(cvkDevice);
+            if (VkFailed(ret)) { 
```

Comment:
I think having it on the one line, like the many other instances, is fine.

Reply:
changed to a one liner

Reply:
This needs to match the other 73 instances.